utils/redis_helper.py

```diff
@@ -0,0 +1,52 @@
+from django.conf import settings
+from utils.redis_client import RedisClient
+from utils.redis_serializers import DjangoModelSerializer
+
+
+class RedisHelper:
+
+    @classmethod
+    def _load_objects_to_cache(cls, key, objects):
```

Comment:
when load objects to the cache, we not only put the new value into the cache, but also we set the expiration time